octodns/manager.py

```diff
@@ -190,9 +190,11 @@ def _config_providers(self, providers_config):
             except KeyError:
                 self.log.exception('Invalid provider class')
                 raise ManagerException(
-                    f'Provider {provider_name} is missing class'
+                    f'Provider {provider_name} is missing class, {provider_config.context}'
```

Comment:
All of the yaml sourced config in manager will only ever be loaded by our custom yaml loaders, `SortEnforcingLoader` or `ContextLoader`, so it's safe to assume they'll be `ContextDict`s.